### ISSUE TITLE
lg-zoom with lg-video crashes when zooming a video on iPhone

### ISSUE THREAD

**Issue by maxsalven** [sachinchoolur/lightGallery#1103]
## Description

If you use both the lgZoom and lgVideo plugins, and have a gallery with both photos and html5 videos, then trying to zoom on a video on an iPhone causes a crash.

### Steps to reproduce

Try to zoom on a video via pinch gesture on an iPhone.

### Expected result

It shouldn't do anything

### Actual result

It crashes

### Additional context

v 2.1.8

```
TypeError: undefined is not an object (evaluating '$image.get().offsetWidth')
(anonymous function)
src/plugins/zoom/lg-zoom.ts:394
```

![IMG_5CC13147F359-1](https://user-images.githubusercontent.com/4144910/127701537-5ec50fd9-582a-4be3-9772-c9a2eb26a35a.jpeg)

**Comment by sachinchoolur** [comment]
Hey @maxsalven,

Thank you for the report. Pinch zoom on the HMTL video player is the default behaviour or IOS. 
I'm able to reproduce the JavaScript error. I'll push a fix soon.

**Comment by sachinchoolur** [comment]
@maxsalven The issue has been fixed
Available in version 2.2.0-beta.4
